lib/index.js

```diff
@@ -9,20 +9,13 @@ import Timer from './plugins/timer';
 import Touch from './plugins/touch';
 import a11y from './plugins/a11y';
 
-export let shower; // eslint-disable-line
-
 const dataAttrsOptions = [
     'debug-mode',
     'slides-selector',
 ];
 
 const hasOptions = typeof showerOptions !== 'undefined';
```

Comment:
I am not especially found of passing options via global variable. `<script>` order, `async`, or `defer` attributes may cause Shower to load before setting the options. Another catch: `type=module` will scope the variable and options will be ignored.

My initial thought was to have `shower-options.js` file and `import()` it in Shower core. However, this is not possible via `file:///` pseudo-protocol (at least in Chrome 63).

Having to `import` Shower from where it is downloaded and calling g method on it to configure does not seem like an ergonomic option either.

Data attributes are limited to flat options structure and string values. Also it seems off that I can set every option except `container_selector`, because I have to set `data-*` on container element.

Reply:
Even static `import` does not work via `file:///` protocol. That means that rollup is here to stay. But we still can consume modules from `http://` origins (plugins, for example).

Reply:
How do you like

```html
<script src="js/shower.js" data-options="{
  containerSelector: 'document.currentScript is awesome!',
}"></script>
```
?

Or should we allow configuring Shower without JS/JSON syntax?

Reply:
I’d go for confuguring the most common things via attributes or class names. As for the rest use cases I’d suggest authors to use plugins or use custom `shower.js` builds for their needs. If we’ll see that it’s not enough we could think about it later. I don’t want to invent the whole options system without strong use cases for now.